translator/translate/otel/exporter/awsemf/translator.go

```diff
@@ -93,6 +93,7 @@ func (t *translator) Translate(c *confmap.Conf) (component.Config, error) {
 	if credentialsFileKey, ok := agent.Global_Config.Credentials[agent.CredentialsFile_Key]; ok {
 		cfg.AWSSessionSettings.SharedCredentialsFile = []string{fmt.Sprintf("%v", credentialsFileKey)}
 	}
+	cfg.AWSSessionSettings.RoleARN = agent.Global_Config.Role_arn
```

Comment:
Can you add a unit or integration test for this?

Reply:
I debated if it was worthwhile to add anything, but I think that adding a translation test here is pretty intentional and self documenting so I'm for adding a unit test. I don't know how we mechanize this so that we don't miss this in other exporters though.

Reply:
Added a simple unit test that checks for the roleARN and region, similar to other exporters. I think it is definitely worthwhile to add an integration test for this use case as well